sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogSuite.scala

```diff
@@ -247,7 +247,7 @@ abstract class ExternalCatalogSuite extends SparkFunSuite with BeforeAndAfterEac
     val catalog = newBasicCatalog()
     val tbl1 = catalog.getTable("db2", "tbl1")
     val newSchema = StructType(Seq(
-      StructField("new_field_1", IntegerType),
+      StructField("col1", IntegerType),
```

Comment:
@tejasapatil do you still remember why we update it?

Reply:
yes. This was done because before this PR the test case was removing bucketed columns in the alter operation. We decided to disallow removing bucketing columns and support this if needed in future. 

Here is the discussion that we both had about this : https://github.com/apache/spark/pull/17644#discussion_r114784136

Reply:
Huh. If I revert the change to this test case, it does not fail anymore. This is bad because the table properties still say that its bucketed over `col1` but `col1` is not in the modified table schema. I am taking a look to see what changed.